osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -130,6 +131,17 @@ private void createTicks()
                     var distanceProgress = d / length;
                     var timeProgress = reversed ? 1 - distanceProgress : distanceProgress;
 
+                    var firstSample = Samples.FirstOrDefault();
+                    var sampleList = new List<SampleInfo>();
+
+                    if (firstSample != null)
+                        sampleList.Add(new SampleInfo
+                        {
+                            Bank = firstSample?.Bank,
+                            Volume = firstSample?.Volume ?? 100,

```

Comment:
Stable uses the control point's volume.

Reply:
that actually sounds wrong

Reply:
Yeah I'm not sure about it, depends if you're worried about consistency or not. I'm fine with it as it is.